src/core/msg.rs

```diff
@@ -154,6 +161,18 @@ pub enum ExecuteMsg {
         /// data from a new location, potentially without any Provenance Blockchain interaction, facilitating the process of data
         /// interaction.
         access_routes: Option<Vec<AccessRoute>>,
+        /// An optional parameter that will cause the emitted events to include values that signal
+        /// to any [Object Store Gateway](https://github.com/FigureTechnologies/object-store-gateway)
+        /// watching the events that the verifier should no longer have permission to inspect the
+        /// identified scope's records via fetch routes.
+        ///
+        /// Note: This route uses a unique identifier based on asset type, so if simultaneous access
+        /// grants were sent to the gateway for different asset types' verifications on a singular
+        /// scope, the verifier will retain access via the gateway until all verifications have been
+        /// completed.
+        ///
+        /// This behavior defaults to TRUE.
+        remove_os_gateway_permission: Option<bool>,
```

Comment:
Doesn't it seem like we shouldn't let the verifier control this behavior? Seems like the scope owner should control it... I guess if that were the case we would have to store their requested behavior on the scope attribute until verification was complete and then use that to determine whether to generate the revoke attribute on verification...

Reply:
I agree - gonna DM you on this one because I have many thoughts here

Reply:
Okee - got these changes added